src/Sylius/Component/Core/Model/ProductReviewerInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Model;
+
+use Sylius\Component\Review\Model\ReviewerInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface ProductReviewerInterface extends ReviewerInterface
```

Comment:
The same with this, this is only a marker of review for specific resource (``Product`` in that case), so we can use only reviews for this resource.